src/meta/src/stream/stream_graph.rs

```diff
@@ -235,74 +198,8 @@ impl StreamActorBuilder {
         });
     }
 
-    /// Build an actor from given information. At the same time, convert local actor id to global
-    /// actor id.
-    pub fn seal(&mut self, actor_id_offset: u32, actor_id_len: u32) {
```

Comment:
This function is only used to turn local actor IDs into global ones, which is unnecessary now.